src/style.css

```diff
@@ -0,0 +1,20 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star--active {
```

Comment:
instead modifier for  .stars__star--active modifier use .stars-1 ... .stars-5 modifiers and `:nth-child` pseudo class